### ISSUE TITLE
Public, protected, etc. not highlighted in argument list.

### ISSUE THREAD

**Issue by rangeoshun** [tcarlsen/language-typescript-grammars-only#8]
Public, protected, etc. not highlighted in argument list.

``` typescript
class Foo {
  constructor (
    public type: number
  ) {
    // Things...
   }
  }
}

```

**Comment by tcarlsen** [comment]
fixed in `1.4.0`

**Comment by rangeoshun** [comment]
Thanks! Very cool! :)
